gen/generator.go

```diff
@@ -103,6 +103,7 @@ var templateFuncs = template.FuncMap{
 }
 
 // Global variables used for unique resource name setting based on label from meta data
```

Comment:
This comment is not at the right place or need to be updated.

Reply:
ok